File: 229-majority-element-ii/majority-element-ii.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        nums_counter = Counter(nums)

        target = len(nums) / 3

        result = []

        for num in nums_counter:
            if nums_counter[num] > target:
                result.append(num)

        
        return result

File: 229-majority-element-ii/majority-element-ii.go
func majorityElement(nums []int) []int {
    var result []int

    numCounter := make(map[int]uint)

    for _, num := range nums{
        numCounter[num]++
    }
    for num, count := range numCounter{
        if float64(count) > float64(len(nums) / 3){
            result = append(result, num)
        }
    }
    return result 
}